.github/workflows/testAndPublish.yml

```diff
@@ -159,9 +159,19 @@ jobs:
       run: ci/scripts/setBuildVersionVars.ps1
     - name: Set scons args
       run: ci/scripts/setSconsArgs.ps1
-    - name: Prepare source code for launcher
+    - name: Install SignPath
+      if: ${{ env.feature_signing }}
+      run: Install-Module -Name SignPath -Force
+    - name: Create unsigned launcher
+      if: ${{ !(github.event_name == 'push' && env.feature_signing) }}
       shell: cmd
       run: scons %sconsOutTargets% %sconsArgs% ${{ !runner.debug && '--all-cores' || '-j1'}}
+    - name: Create signed launcher
+      if: ${{ github.event_name == 'push' && env.feature_signing }}
```

Comment:
Will this work when creating a release tag, I wonder? It somehow occurs to me that the push event is in one way too restrictive and in another way too permissive.

Reply:
yes, pushing to tags counts as a push event. The behaviour of this matches AppVeyor